templates/http-csharp/metadata/spin-template.toml

```diff
@@ -6,3 +6,7 @@ description = "HTTP request handler using C# (EXPERIMENTAL)"
 project-description = { type = "string", prompt = "Project description", default = "" }
 http-base = { type = "string", prompt = "HTTP base", default = "/", pattern = "^/\\S*$" }
 http-path = { type = "string", prompt = "HTTP path", default = "/...", pattern = "^/\\S*$" }
+
+[[custom_filters]]
+name = "dotted_pascal_case"
+wasm = "dotted_pascal_case_filter.wasm"
```

Comment:
does this `dotted_pascal_case_filter.wasm` file need to be added under `metadata/filters`?

Reply:
Yikes, thanks for the catch.  Looks like the .gitignore is fouling it up.